protocol/0070-MKTD-market-decimal-places.md

```diff
@@ -12,6 +12,13 @@ This document aims to outline how we are to handle the decimal places of a given
 
 It is possible to configure a market where orders can only be priced in increments of a specific size. This is done by specifying a different (smaller) number of decimal places than its settlement asset supports. Simply put: a market that settles in GBP can be configured to have 0 decimal places, in which case the price levels on the orderbook will be at least separated by £1, rather than the default penny.
 
+In order to ensure we are able to represent the smallest trade at the smallest price in quote asset decimals. We want
```

Comment:
This is also only for spot. 

For derivatives the issue is different: you want to make sure that the smallest MTM cashflow can be represented in asset decimals.